rdr_service/storage.py

```diff
@@ -225,6 +225,36 @@ def __enter__(self):
     def __exit__(self, *exc):
         self.close()
         return False
+        
+    def __iter__(self):
```

Comment:
This function conflicts with the same function definition at line 177.  The one at line 177 probably needs to be removed.

Reply:
Never mind, saw you removed it in a later commit.